### DISCUSSION TITLE
Build Issue with the Azure Devops Pipeline

### DISCUSSION THREAD

**Discussion by subhashfission** [microsoft/dicom-server#782] [Q&A]
Getting Below Issuw while Build the Project From pipeline which is provided in the build 

`/home/vsts/work/1/s/src/Microsoft.Health.Dicom.Client/obj/Release/net5.0/Microsoft.Health.Dicom.Client.AssemblyInfo.cs(17,59): error CS7035: The specified version string does not conform to the recommended format - major.minor.build.revision [/home/vsts/work/1/s/src/Microsoft.Health.Dicom.Client/Microsoft.Health.Dicom.Client.csproj]`

`/home/vsts/work/1/s/src/Microsoft.Health.Dicom.Client/obj/Release/net5.0/Microsoft.Health.Dicom.Client.AssemblyInfo.cs(21,55): error CS7034: The specified version string does not conform to the required format - major[.minor[.build[.revision]]] [/home/vsts/work/1/s/src/Microsoft.Health.Dicom.Client/Microsoft.Health.Dicom.Client.csproj]`

Is there any solution for this

**Comment by jovinson-ms** [comment]
Hi @subhashfission, sorry we didn't get see your question earlier! Is this still an issue for you? We are not seeing any build issues like what you provided.
